env/site.yml

```diff
@@ -44,8 +44,10 @@ engines:
           timecode: automatic
           frame: 1001
     menu_overrides:
+      SG Review:
+        - {app_instance: tk-multi-importcut, name: Import Cut}
       Tools:
-      - {app_instance: tk-multi-pythonconsole, name: PTR Python Console...}
+        - {app_instance: tk-multi-pythonconsole, name: PTR Python Console...}
```

Comment:
I'm not sure if the indentation is correct. I believe it has two extra whitespace. Can you confirm if this is working as expected? If so I can ✅ 

```suggestion
      - {app_instance: tk-multi-pythonconsole, name: PTR Python Console...}
```

Reply:
I think the indent will work and looks better this way. But yes, please confirm it works.